src/containers/App/App.js

```diff
@@ -12,26 +13,29 @@ const App = (props) => {
 
 
   return (
-    <div>
+    <>
       {props.validatedUser && <Header />}
-      <Switch>
-        <Route
-          path="/"
-          exact
-          render={() => <Home />}
-        />
-        <Route
-          path="/configure"
-          exact
-          render={() => <Configure />}
-        />
-        <Route
-          path="/historical"
-          exact
-          render={() => <Historical />}
-        />
-      </Switch>
-    </div>
+      <div className="background-container">
+        <Ecotip />
+        <Switch>
+          <Route
+            path="/"
+            exact
+            render={() => <Home />}
+          />
+          <Route
+            path="/configure"
+            exact
+            render={() => <Configure />}
+          />
+          <Route
+            path="/historical"
+            exact
+            render={() => <Historical />}
+          />
+        </Switch>
+      </div>
+    </>
```

Comment:
Wrapping this in a div for background-container is problematic for reasons I don't entirely understand yet, and I'm open to changing this out in this PR!

Reply:
Yea, this is a good addition because it's the div we will want to display the background on but it does result in a few minor issues with styling from other pages.  I think we can mostly fix it by changing the `background-container` classname on the other divs that it's currently on, and only moving all the non-background related rules into that new classname.

Reply:
There's a lot of tricky situations with Switch that even we don't understand.  Good on you for placing this in the right spot.  We will get the styling figured out!